schutzbot/ci_details.sh

```diff
@@ -42,19 +42,22 @@ echo "List of installed packages:"
 rpm -qa | sort
 echo "------------------------------------------------------------------------------"
 
-if rpm --quiet -q python36; then
-    echo -e "\n FAIL: python36 is installed, see #794 ..."
-    exit 1
-else
-    echo -e "\n PASS: python36 not insalled"
+# gcp runners don't use cloud-init and some of the images have python36 installed
```

Comment:
- Which of the GCP images have python36 installed? 
- Do they have another installation of python, e.g. platform-python ? What version is it?

Reply:
I believe it's all el-8. So centos-8 and any rhel-8... They have both `platform-python` and `python36` ... As an example you can see this job output where I've let it list all the packages: https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/jobs/3776825895#L1028 

there is the same runner on aws in that pipeline and it's just missing the `python36` package from what I can tell. They bot have `platform-python-3.6.8` though. I don't even know what the original issue with this was as the Github issue has link to Jenkins which no longer exists. It doesn't seem to cause problems from what I can tel...